docs/_advanced/error_handling.md

```diff
@@ -6,9 +6,9 @@ order: 1
 ---
 
 <div class="section-content">
-If an error occurs in a listener, it’s recommended you handle it directly. However, there are cases where errors may be triggered after your listener has already returned (such as in `say()`, `respond()`, or if you don't call `ack()` when it's required). By default, these errors will be logged to the console. To handle them yourself, you can attach a global error handler to your app with the `error(fn)` method.
+*Note: Since `Bolt@2.x`, error handling has improved! View the [migration guide for V2](https://slack.dev/bolt/tutorial/migration-v2) to learn more.*
```

Comment:
i think the `Bolt@2.x` naming is new and possibly unfamiliar. can we just say v2? if we're going to be specific enough to put it in a code format, then it should probably be the package/version name: `@slack/bolt@v2.0.0`

Reply:
```suggestion
*Note: Since v2, error handling has improved! View the [migration guide for V2](https://slack.dev/bolt/tutorial/migration-v2) to learn about the changes.*
```